.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
is it necessary to add settings.json?

Reply:
as the project is cloned from the git and this file is with that clone now the file is deleted and project is pushed